templates/gitlab-manager-runner.yaml

```diff
@@ -291,6 +292,7 @@ Resources:
                       MachineDriver = "amazonec2"
                       MachineName = "gitlab-docker-machine-%s"
                       MachineOptions = [
+                        "amazonec2-ami=ami-08d4ac5b634553e16"
```

Comment:
This should be parameterized I think, will check with Jon.

Reply:
This will set the AMI for the spawn runners.
If you want to set a specific AMI for the Manager itself, set the `ManagerImageId` param in the `iatlas-runner.yaml` config (both staging and prod).

Reply:
And, yes, you could parameterize this so that it is easier to update moving forward.

Reply:
seems like you are missing a comma at the end?  and +1 for parameterizing it.